frontend/app/services/lookup-service.server.ts

```diff
@@ -323,46 +328,71 @@ function createLookupService() {
   }
```

Comment:
`getAllPreferredCommunicationMethods()` is also called within the `/apply` flow and should be logged.